admin_guide/diagnostics_tool.adoc

```diff
@@ -115,3 +115,78 @@ A client with *cluster-admin* access available (for any user, but only the
 current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
+
+[[additional-cluster-health-checks]]
+== Additional Diagnostic Checks via Ansible
+
+// TODO: add link to OCP image once it is available
+
+Some additional diagnostic checks are available through the *openshift-ansible*
+container image. See the image's link:https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[source repository] for usage information.
+
+The following health checks belong to a diagnostic task meant to be run against
+the Ansible inventory file for a deployed {product-title} cluster. They can
+report common problems for the current {product-title} installation.
+
+[[admin-guide-diagnostics-tool-ansible-checks]]
+.Diagnostic Checks
+[options="header"]
+|===
+
+|Check Name |Purpose
+
+|`ovs_version`
+|This check ensures that a host has the correct version of Open vSwitch installed
+for the currently deployed version of {product-title}.
+
+|`kibana`, `curator`, `elasticsearch`, `fluentd`
+|This set of checks verifies that Elasticsearch, Fluentd, and Curator pods have
+been deployed and are in a `running` state, and that a connection can be
+established between the control host and the exposed Kibana URL. These checks
+will only run if the `openshift_hosted_logging_deploy` inventory variable is set
+to `true`, to ensure that they are executed in a deployment where a logging
+stack has been deployed.
+
+|`etcd_imagedata_size`
+|This check measures the total size of {product-title} image data in an etcd
+cluster. The check fails if the calculated size exceeds a user-defined limit. If
+no limit is specified, this check will fail if the size of image data amounts to
+`50%` or more of the currently used space in the etcd cluster.
```

Comment:
Do we need the ticks around 50%? Seems like more of a descriptive number than a value or a setting.